pkg/cluster/delete.go

```diff
@@ -451,7 +451,7 @@ func (m *manager) Delete(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		err = oidcbuilder.DeleteOidcFolder(ctx, env.OIDCBlobDirectoryPrefix+m.doc.ID, azBlobClient)
```

Comment:
Nit: `OIDCBlobDirectoryPrefix` can be removed from `pkg/env/env.go` since it's only references are removed in this PR.